NextGenSoftware.Holochain.HoloNET.Client.Core/HoloNETClient.cs

```diff
@@ -456,7 +456,7 @@ public async Task StartHolochainConductorAsync()
                     if (Config.HolochainConductorToUse == HolochainConductorEnum.HcDevTool)
                     {
                         //_conductorProcess.StartInfo.Arguments = "sandbox run 0";
-                        _conductorProcess.StartInfo.Arguments = $"hc sandbox generate {Config.FullPathToCompiledHappFolder}";
+                        _conductorProcess.StartInfo.Arguments = $"sandbox generate {Config.FullPathToCompiledHappFolder}";
```

Comment:
Ooops! Yes you are right, missed that one, further down it is correct but missed this one, thanks for the correction! ;-)